tests/integration/bigquery.spec.ts

```diff
@@ -360,6 +360,29 @@ suite("@dataform/integration/bigquery", ({ before, after }) => {
       expect(increment[increment.length - 2].statement).to.equal(table.postOps[0]);
       expect(increment[increment.length - 1].statement).to.equal(table.postOps[1]);
     });
+
+    test("contextual pre and post ops", async () => {
+      const table: dataform.ITable = {
+        type: "incremental",
+        query: "query",
+        preOps: ["preOps"],
+        incrementalQuery: "incrementalQuery",
+        postOps: ["postOps"],
+        target: { schema: "", name: "", database: "" }
+      };
+
+      const bqadapter = new BigQueryAdapter({ warehouse: "bigquery" }, "1.6.12");
+
+      const refresh = bqadapter
+        .publishTasks(table, { fullRefresh: true, useContextualOps: true }, { fields: [] })
+        .build();
+      expect(refresh.length).to.equal(1);
```

Comment:
can we change these to assert equality against the actual query string?

Reply:
Sure, I was avoiding doing that in order to keep the tests more flexible